client-api/pom.xml

```diff
@@ -16,6 +16,26 @@
       <version>3.0.2</version>
       <scope>provided</scope>
     </dependency>
+    <dependency>
+      <groupId>org.sonarsource.sonarqube</groupId>
+      <artifactId>sonar-plugin-api</artifactId>
```

Comment:
I think that adding a dependency on sonar-plugin-api just for the RuleKey class is overkill.